meerk40t/core/elements.py

```diff
@@ -2874,15 +2874,59 @@ def my_sign(x):
             # Returns +1 for positive figures, -1 for negative and 0 for Zero
             return bool(x > 0) - bool(x < 0)
 
-        from meerk40t.svgelements import Line
+        def list_subpath_bounds(obj):
```

Comment:
The API for Path objects basically treats them like a list anyway. You don't need to dig into the class and access the `._segements` attribute. This bit of code seems like it could use some `path.as_subpath()` code.

Reply:
Did this as I needed to reshuffle the segments anyway, so I assume at that point I would need _segments